### PR TITLE
Add 'Update the Kubernetes version' to 'Building' site doc

### PR DESCRIPTION
**Pull request by ilya-zuyev**

### CODE REVIEW

File: site/content/en/docs/contrib/releasing/binaries.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ description: >
 * Two minikube repos checked out locally:
   * Your personal fork
   * Upstream  
+  
+## Update the Kubernetes version
+
+* Run `make update-kubernetes-version` from your local upstream repo copy
+* If any files are updated, combine these changes with the Release Notes PR
```
Comment (medyagh) [inline site/content/en/docs/contrib/releasing/binaries.md:19]: this would need to be a separate PR. so we can test the health of the new kubernets version

**Review by medyagh — CHANGES_REQUESTED** [review]

File: site/content/en/docs/contrib/releasing/binaries.md (hunk 1 above)
Comment (ilya-zuyev) [reply site/content/en/docs/contrib/releasing/binaries.md]: fixed

**Review by medyagh — APPROVED** [review]
